src/runtime/cuda/cuda_hardware_manager.cpp

```diff
@@ -164,7 +164,7 @@ bool cuda_hardware_context::has(device_support_aspect aspect) const {
   case device_support_aspect::global_mem_cache_read_only:
     return false;
     break;
-  case device_support_aspect::global_mem_cache_write_only:
+  case device_support_aspect::global_mem_cache_read_write:
     return false;
```

Comment:
Now that the semantics has changed from write-only to read-write - I suspect we will have to return true for CUDA, since I think CUDA hardware has read-write cache. But better check to be sure ;)